src/Majora/Bundle/FrameworkExtraBundle/DependencyInjection/Compiler/LoaderCompilerPass.php

```diff
@@ -87,6 +88,15 @@ function ($property) { return $property->getName(); },
                         ))
                     ;
                 }
+
+                if (!isset($attributes['alias'])) {
+                    throw new \InvalidArgumentException(sprintf('Alias not define for "%s" loader.', $loaderId));
```

Comment:
You will introduce a bc break here, i think i should be better to do nothing if any alias.
Or trigger a deprecation call, saying an alias is required.